fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/ContentObject.java

```diff
@@ -89,12 +90,13 @@ public FileObject addDescription(InputStream sourceMdStream, String sourceProfil
         FileObject fileObj = createFileObject();
 
         BinaryObject orig = fileObj.addOriginalFile(sourceMdStream, null, "text/plain", null, null);
-        repoObjFactory.createProperty(orig.getPid(), Cdr.hasSourceMetadataProfile, sourceProfile);
-        repoObjFactory.createRelationship(orig.getPid(), RDF.type, Cdr.SourceMetadata);
-        repoObjFactory.createRelationship(pid, PcdmModels.hasRelatedObject, fileObj.getResource());
+        URI metadataUri = RepositoryPaths.getMetadataUri(orig.getPid());
```

Comment:
Is this being used?

Reply:
Nope, good catch, I think that was left over from an earlier iteration. Might make same changes to the way descriptions are stored anyways after talking to anna.

Reply:
Were the added changes going to be after this is merged?